x-pack/solutions/observability/plugins/apm/public/components/fleet_integration/apm_policy_form/settings_definition/agent_authorization_settings.ts

```diff
@@ -57,7 +57,7 @@ export function getAgentAuthorizationSettings(): SettingsRow[] {
       ],
     },
     {
-      type: 'text',
+      type: 'secret',
```

Comment:
Changing this value immediately has the effect of using the "fallback" component that renders a `EUiFieldPassword`, but it doesn't affect the functionality - secret_token keeps being saved as plain text. 

I think that this is acceptable but let me know if it's better to change it later when the new APM version with `secret: true` will be released.